client/src/components/MultiStep/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Box } from '@material-ui/core';
+import BasicUserInfo from '../signUpForm/BasicUserInfo';
+import GenderSelection from '../signUpForm/GenderSelection';
+import PhysicalCharacteristics from '../signUpForm/PhysicalCharacteristics';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+  },
+  button: {
+    marginRight: theme.spacing(1),
+  },
+  instructions: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+export default function HorizontalLinearStepper() {
```

Comment:
It might be a good idea to give this file and component a more descriptive name, like `SignupFormSteps` or something like that.  Its current name sounds pretty generic, as if it's a component that is being reused in many different places.